main/modules/menus/file.js

```diff
@@ -66,18 +66,6 @@ function buildFileMenu(fileURL, getTrialInfo) {
             }
           },
         },
-        {
```

Comment:
This looks like a mistake.
We shouldn't remove this if there's no alternative for saving a project with a new name via the file menu.
@cameronsutter, can you please comment on this?

Reply:
@Quiescent reverted back 😅

Reply:
it looks like that was incorrect, but it seems to be reverted, so 👍